app/views/admin/users/index.html.slim

```diff
@@ -35,5 +35,3 @@
                   - user.cases.opened.each do |kase|
                     li
                       = link_to kase.number, case_path(kase.id)
-
-= paginate @users
```

Comment:
Cant we just leave the pagination and just increase the number of of results? Just thinking listing over 700+ users and all their cases on a single page might put the server under a bit of pressure

Reply:
Sure! Not a problem! Might be nice to keep it fairly high though, I'm just thinking of using find on the page as a quick way to track down users.

How many do you think? 100? 200? 100000000000000 ;)

Reply:
1million should just about do it :P 

Feature test?  ;)